playbook/workshops/contrib/ajar/README.md

```diff
@@ -234,4 +234,174 @@ You've finished Part 1!
 
 ![](img/celebrate.gif)
 
-# More coming soon...
+
+## Part 2
+
+Welcome to part 2 of the Ajar.io workshop!
+
+To recap, in [part 1](#part-1) you learned about:
+
+- **JavaScript variables**: used to store data values. Declared using the `var`
+  keyword.
+- **Drawing on the canvas**: the HTML `<canvas>` element is used to draw
+  graphics, on the fly, via scripting (usually JavaScript). You obtain the 
+  drawing context of the `<canvas>` element by calling the _function_ (more on
+  what that means today) `getContext("2d")` of the `<canvas>` element. You use
+  the drawing context to draw shapes on the canvas.
+
+In this part you will learn about **functions**.
+
+### What are functions?
+
+In part 1, you wrote all your code outside of any functions. This means that
```

Comment:
It seems like the word `any` can be used with either plural or singular nouns.

Source: http://english.stackexchange.com/a/113005